tests/common/schema/test_inference.py

```diff
@@ -373,12 +373,14 @@ def __call__(self) -> Any:
     assert exc_val.value.coerced_value == "no double"
 
 
-def test_corece_new_null_value(schema: Schema) -> None:
+def test_coerce_new_null_value(schema: Schema) -> None:
     row = {"timestamp": None}
     new_row, new_table = schema.coerce_row("event_user", None, row)
+    # No new rows, but new column in schema
     assert "timestamp" not in new_row
-    # columns were not created
-    assert new_table is None
+    assert "data_type" not in new_table["columns"]["timestamp"]
+    assert new_table["columns"]["timestamp"]["nullable"] is True
+    assert new_table["columns"]["timestamp"]["x-normalizer"]["seen-null-first"] is True
```

Comment:
please also test if we raise when a column is supposed to be not null. we should raise. see above. `_dlt_id` is supposed to be not null. (schema should contain this):
```
"not_null": [
            "_dlt_id",
            "_dlt_root_id",
            "_dlt_parent_id",
            "_dlt_list_idx",
            "re:^_dlt_load_id$",
        ],
```)